lte/gateway/c/oai/lib/s6a_proxy/S6aClient.h

```diff
@@ -83,7 +83,7 @@ class S6aClient : public GRPCReceiver {
   S6aClient();
   static S6aClient& get_instance();
   std::unique_ptr<feg::S6aProxy::Stub> stub_;
-  static const uint32_t RESPONSE_TIMEOUT = 3;  // seconds
+  static const uint32_t RESPONSE_TIMEOUT = 10;  // seconds
```

Comment:
Considering the UE will re-try Attach in around 15 seconds, if it does not receive an Attach Accept, shouldn't we fail faster?

Reply:
@ssanadhya the rationale for this is that if there is a db lock issue and if it is a transient thing to see large latency in these calls, the rest of the procedures would still have meaningful time budget to complete the remaining steps (e.g., the remaining ~5sec). If this is not a transient thing and timeouts still happen, then prolonging the timeout can help push back the attach request of UEs relieving the flash crowds a bit. Does this make sense?

Reply:
Makes sense! So in case of a severe failure, the UEs will be stalled for 10s before re-trying an attach, slowing down the request flow into subscriberdb, correct?

Reply:
yes...